src/containers/Login/component.js

```diff
@@ -19,44 +21,50 @@ const Login = ({
             sm={{ span: 10, offset: 1 }}
             lg={{ span: 8, offset: 2 }}
           >
-            <Form onFinish={login}>
+            <Form onSubmit={handleSubmit}>
               <Typography.Title>The Movie DB</Typography.Title>
               {errorTxt && (
-              <Alert
-                type="error"
-                message={errorTxt}
-              />
+                <Alert
+                  type="error"
+                  message={errorTxt}
+                />
               )}
-              <Form.Item name="username">
-                <Input
-                  prefix={(
+              <Field
+                name="username"
+                validateStatus={(username && touched.username) ? 'error' : null}
+                help={touched.username && username}
```

Comment:
it would be better to handle it inside `FormField` component. there you have `field` and `form` objects. so you can take `name` propoerty from `filed` and `tocuhed` property from `form` and than do this `touched[name]`. so you will get DRY solution